en/docs/references/default-ports-of-wso2-products.md

```diff
@@ -0,0 +1,175 @@
+# Default Ports of WSO2 Products
+
+This page describes the default ports that are used for each WSO2
+product when the port offset is 0. If you are running multiple WSO2
+products on the same server, you must set the
+`        offset       ` value in the 
+`         <IS_HOME>/repository/conf/deployment.toml      ` file to a
+different value for each product so that there are no port conflicts.
+
+``` toml
+[server]
+offset = ""
+```
+
+For example, if you are running two WSO2 products on the same server,
+and you set the port offset to 1 in one product and 2 in the second
+product, the management console port will be changed from the default of
+9443 to 9444 in the first product and to 9445 in the second product. See
+[here](#changing-the-offset-for-default-ports) for more information on
+changing the offset.
+
+## Common ports
+
+The following ports are common to all WSO2 products that provide the
+given feature. Some features are bundled in the WSO2 Carbon platform
+itself and therefore are available in all WSO2 products by default.
+
+## Management console ports
+
+WSO2 products that provide a management console use the following
+servlet transport ports:
+
+-   9443 - HTTPS servlet transport (the default URL of the management
+    console is https://localhost:9443/carbon )
+-   9763 - HTTP servlet transport
+
+### LDAP server ports
+
+Provided by default in the WSO2 Carbon platform.
+
+-   10389 - Used in WSO2 products that provide an embedded LDAP server
+
+### KDC ports
+
+-   8000 - Used to expose the Kerberos key distribution center server
+
+### JMX monitoring ports
+
+WSO2 Carbon platform uses TCP ports to monitor a running Carbon instance
+using a JMX client such as JConsole. By default, JMX is enabled in all
+products. To disable it, see [Disabling JMX for the server](../../../deploy/monitor/jmx-based-monitoring#disabling-jmx-for-the-server).
```

Comment:
add TODO placeholder for outdated links?

Reply:
Added TODO for pages that I couldn't find. Since these are pages that we are going to dump as it is for now, added them as it is.